go/saml-client.go

```diff
@@ -66,7 +66,11 @@ func (s SamlClient) GetSPParameters(ctx context.Context, req *saml.GetSPParamete
 	defer conn.Close()
 	client := saml.NewSamlConfigurationServiceClient(conn)
 
-	return client.GetSPParameters(callProperties.Ctx, req, callProperties.CallOptions...)
+	response, err := client.GetSPParameters(callProperties.Ctx, req, callProperties.CallOptions...)
+	if err != nil {
+		return nil, NewSdkAPIError(err, saml.SamlConfigurationService_GetSPParameters_FullMethodName, "aaa")
```

Comment:
"aaa" another const in here. re-export the SamlConfigurationServiceGetSPParamsRPC